MediaGalleryRenditionsApi/Api/ConfigInterface.php

```diff
@@ -6,7 +6,7 @@
 
 declare(strict_types=1);
 
-namespace Magento\MediaGalleryRenditionsApi\Model;
+namespace Magento\MediaGalleryRenditionsApi\Api;
```

Comment:
The `ConfigInterface` is not a service so it's better to keep it in the `Model` namespace (it is still the API module).

Reply:
Okay, But I moved into the `Api` namespace because of the inconsistency with the other module. If you check the other `ConfigInterface` in the repository they are under the `Api` namespace so should I move the other `ConfigInterface` to the `Model` namespace?

Reply:
ConfigInterface is not designed to be used outside of the MediaGalleryRenditions module so it's not part of actual module's API, but it still can be customized (preference, plugin, etc)